frontend/src/app/shop/order-menu/order-menu.component.html

```diff
@@ -106,14 +112,27 @@
               </div>
             </div>
 
-            <div class="flex gap-1 items-center">
+            <!-- <div class="flex gap-1 items-center">
               <img
                 [src]="product.image"
                 [alt]="product.name"
                 class="w-[65px] h-[65px]" />
               <div>
                 {{ product.description }}
               </div>
+            </div> -->
+
+            <!-- Easter Egg product display logic -->
+            <div class="flex gap-1 items-center">
+              <ng-container *ngIf="product.description.toLowerCase().includes('easter egg')">
+                <mat-icon class="w-[65px] h-[65px]">egg</mat-icon>
+              </ng-container>
+              <ng-container *ngIf="!product.description.toLowerCase().includes('easter egg')">
+                <mat-icon class="w-[65px] h-[65px]">inventory_2</mat-icon>
```

Comment:
instead css in the html, put entry into order-menu.comp.scss:
mat-icon.my-icon-class {
  width: 65px; 
  height: 65px;
}

Reply:
Sorry, i am revoking this comment. Its legit tailwind